web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -70,20 +71,21 @@
                         class="access-grants__flows-area__learn-button"
                     />
                     <VButton
-                        label="Create Access Grant"
+                        label="Create S3 Credentials"
```

Comment:
Should this say `S3 Compatible Credentials` or similar? Because, technically they are not S3 credentitals.